### ISSUE TITLE
comparefiles utility returns 0 if target file doesn't exist

### ISSUE THREAD

**Issue by hsolbrig** [biolink/linkml#9]
The compare files utility should return a fail if one or both of the files to be compared does not exist

**Comment by joeflack4** [comment]
Seems like an easy starter issue!

**Comment by joeflack4** [comment]
@cmungall I put a fix in: https://github.com/linkml/linkml-runtime/pull/55

I haven't actually used `click` before. Been long wanting to try that. I was trying to fix this the old fashioned way but saw that there are special checks that click can do for you, so I used that, and then did a little refactoring otherwise.

**Comment by cmungall** [comment]
yes, we love click, makes it easier to implement guidelines here: https://clig.dev/
